lib/delivery_boy/config.rb

```diff
@@ -7,6 +7,7 @@ class Config < KingKonf::Config
     # Basic
     list :brokers, items: :string, sep: ",", default: ["localhost:9092"]
     string :client_id, default: "delivery_boy"
+    string :log_level, default: nil
```

Comment:
Ruby's `Logger` has `debug` level by default.
`default: nil` guarantees that it will not break anything.

Reply:
Is this needed? You can alter the logger by using a different one with any log level you want the way it is now.

Reply:
Ok, probably not, but in my situation, it seems to be a solution.

I have a Rails-app, in which I use delivery_boy, I had alter default logger, but I also need to set a log level to it. So the first approach I tried is to add `log_level` to `config/delivery_boy.yml`. But it fails with

```
unknown configuration variable log_level (KingKonf::ConfigError)
```

:confused: So I think it will be ok to add this option to the code.

I don't want to add another config file just for this option. And don't want to write code like this in `config/initializers/delivery_boy.rb`

```ruby
log_levels = {
  production: :warn,
  staging: :warn,
  ...
}
log_level = log_levels[Rails.env.to_sym]
DeliveryBoy.logger = log_level
```

because configuration must be in one place and this place is `delivery_boy.yml`

Reply:
Another solution as I see is to remove restriction of king_kong gem to have only whitelisted keys in yml-config.

Reply:
@mensfeld WDYT?

Reply:
I still don't like this idea ;) but worth asking @dasch. For me, it seems like a redundant functionality as you can inject a custom initialized logger with any log levle.